AssetManagementTestProject/PageObj/ChangePasswordPage.cs

```diff
@@ -6,64 +6,64 @@ namespace AssetManagementTestProject.PageObj;
 public class ChangePasswordPage : WebDriverAction
 {
     #region CHANGE PASSWORD
-    private string btnNavigationBar = "//div[contains(@class, 'ant-dropdown-trigger cursor-pointer')]";
-    private string tfNewPw = "//input[contains(@id, 'newPassword')]";
-    private string tfOldPw = "//input[contains(@id, 'oldPassword')]";
-    private string btnSaveNewPw = "//button[contains(@type, 'submit')]";
-    private string btnChangePw = "//a[contains(@href, '/change-password')]";
-    private string btnCancel = "//span[contains(text(), 'Cancel')]";
-    private string headerChangePw = "//h1[text()='Change Password']";
-    private string pathChangePw = "change-password";
-    private string textChangePwSuccessfully = "//p[text()='Your password has been changed successfully']";
+    private string _btnNavigationBar = "//div[contains(@class, 'ant-dropdown-trigger cursor-pointer')]";
+    private string _tfNewPw = "//input[contains(@id, 'newPassword')]";
+    private string _tfOldPw = "//input[contains(@id, 'oldPassword')]";
+    private string _btnSaveNewPw = "//button[contains(@type, 'submit')]";
+    private string _btnChangePw = "//a[contains(@href, '/change-password')]";
+    private string _btnCancel = "//span[contains(text(), 'Cancel')]";
+    private string _headerChangePw = "//h1[text()='Change Password']";
+    private string _pathChangePw = "change-password";
+    private string _textChangePwSuccessfully = "//p[text()='Your password has been changed successfully']";
```

Comment:
I've seen all your properties in every PageObj are public. Can you change all these page's properties into public

Reply:
Done